src/graphql/fragments/colony.graphql

```diff
@@ -100,6 +98,12 @@ fragment ColonyMetadata on ColonyMetadata {
   }
 }
 
+fragment ColonyBalances on ColonyBalances {
+  items {
+    ...ColonyBalance
+  }
+}
+
```

Comment:
If we're adopting this pattern, then I'd argue this should be done for all fragments (jn this file, at least).

 Ie: `fundsClaims` as well

Reply:
The reason for adopting this is that we used to have an incorrect casting here: https://github.com/JoinColony/colonyCDapp/pull/320/files#diff-7b7b3e96bf501571f1ccd3bfff0c483a4dd74ac8260dfd344b3e52010d1f134aL47
`ColonyBalances` is a GraphQL type, not what we query for.

To be honest, I don't see a point in wrapping the balance data in `items` at all, it feels like it could be just directly returned by the query as an array `[ColonyBalance]`. And we'd have one less type to worry about. Was your intention here to make it consistent with arrays `items` returned by the automatically generated queries?